# References

[^cite_amir13]:
    Amir _et al._ (2013),
    _viSNE enables visualization of high dimensional single-cell data and reveals phenotypic heterogeneity of leukemia_,
    [Nature Biotechnology](https://doi.org/10.1038/nbt.2594).

[^cite_blondel08]:
    Blondel _et al._ (2008),
    _Fast unfolding of communities in large networks_,
    [J. Stat. Mech.](https://doi.org/10.1088/1742-5468/2008/10/P10008).

[^cite_chippada18]:
    _ForceAtlas2 for Python and NetworkX_,
    [GitHub](https://github.com/bhargavchippada/forceatlas2).

[^cite_coifman05]:
    Coifman _et al._ (2005),
    _Geometric diffusions as a tool for harmonic analysis and structure definition of data: Diffusion maps_,
    [PNAS](https://doi.org/10.1038/nmeth.3971).

[^cite_csardi06]:
    Csardi _et al._ (2006),
    _The igraph software package for complex network research_,
    [InterJournal Complex Systems](http://igraph.org).

[^cite_fruchterman91]:
    Fruchterman & Reingold (1991),
    _Graph drawing by force-directed placement_,
    [Software: Practice & Experience](http://doi.org:10.1002/spe.4380211102).

[^cite_haghverdi15]:
    Haghverdi _et al._ (2015),
    _Diffusion maps for high-dimensional single-cell analysis of differentiation data_,
    [Bioinformatics](https://doi.org/10.1093/bioinformatics/btv325).

[^cite_haghverdi16]:
    Haghverdi _et al._ (2016),
    _Diffusion pseudotime robustly reconstructs branching cellular lineages_,
    [Nature Methods](https://doi.org/10.1038/nmeth.3971).

[^cite_islam11]:
    Islam _et al._ (2011),
    _Characterization of the single-cell transcriptional landscape by highly multiplex RNA-seq_,
    [Genome Research](https://doi.org/10.1101/gr.110882.110).

[^cite_jacomy14]:
    Jacomy _et al._ (2014),
    _ForceAtlas2, a Continuous Graph Layout Algorithm for Handy Network Visualization Designed for the Gephi Software_
    [PLOS One](https://doi.org/10.1371/journal.pone.0098679).

[^cite_johnson07]:
    Johnson, Li & Rabinovic (2007),
    _Adjusting batch effects in microarray expression data using empirical Bayes methods_,
    [Biostatistics](https://doi.org/10.1093/biostatistics/kxj037).

[^cite_lambiotte09]:
    Lambiotte _et al._ (2009)
    _Laplacian Dynamics and Multiscale Modular Structure in Networks_
    [arXiv](https://arxiv.org/abs/0812.1770).

[^cite_leek12]:
    Leek _et al._ (2012),
    _sva: Surrogate Variable Analysis. R package_
    [Bioconductor](https://doi.org/10.18129/B9.bioc.sva).

[^cite_levine15]:
    Levine _et al._ (2015),
    _Data-Driven Phenotypic Dissection of AML Reveals Progenitor--like Cells that Correlate with Prognosis_,
    [Cell](https://doi.org/10.1016/j.cell.2015.05.047).

[^cite_maaten08]:
    Maaten & Hinton (2008),
    _Visualizing data using t-SNE_,
    [JMLR](http://www.jmlr.org/papers/v9/vandermaaten08a.html).

[^cite_mcinnes18]:
    McInnes & Healy (2018),
    _UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction_,
    [arXiv](https://arxiv.org/abs/1802.03426).

[^cite_pedersen12]:
    Pedersen (2012),
    _Python implementation of ComBat_
    [GitHub](https://github.com/brentp/combat.py).

[^cite_pedregosa11]:
    Pedregosa _et al._ (2011),
    _Scikit-learn: Machine Learning in Python_,
    [JMLR](http://www.jmlr.org/papers/v12/pedregosa11a.html).

[^cite_traag17]:
    (2017),
    _Louvain_,
    [GitHub](https://doi.org/10.5281/zenodo.35117).

[^cite_traag18]:
    Traag _et al._ (2018),
    _From Louvain to Leiden: guaranteeing well-connected communities_
    [arXiv](https://arxiv.org/abs/1810.08473).

[^cite_weinreb17]:
    Weinreb _et al._ (2016),
    _SPRING: a kinetic interface for visualizing high dimensional single-cell expression data_,
    [bioRxiv](https://doi.org/10.1101/090332).

[^cite_wolf18]:
    Wolf _et al._ (2018),
    _Scanpy: large-scale single-cell gene expression data analysis_,
    [Genome Biology](https://doi.org/10.1186/s13059-017-1382-0).

[^cite_wolf19]:
    Wolf _et al._ (2019),
    _PAGA: Graph abstraction reconciles clustering with trajectory inference through a topology preserving map of single cells._
    [Genome Biology](https://doi.org/10.1186/s13059-019-1663-x),
    [bioRxiv](https://doi.org/10.1101/208819).

[^cite_zunder15]:
    Zunder _et al._ (2015),
    _A continuous molecular roadmap to iPSC reprogramming through progression analysis of single-cell mass cytometry_,
    [Cell Stem Cell](https://doi.org/10.1016/j.stem.2015.01.015).
